packages/internals/funjucks/src/filters/await.test.js

```diff
@@ -0,0 +1,45 @@
+const {expect} = require('../../../../../test/helpers');
+const factory = require('./await');
+
+describe('await', function () {
+  it('exports a function', function () {
+    expect(factory).to.be.a('function');
+  });
+
+  it('returns an object with the expected properties', function () {
+    const filter = factory();
+    expect(filter.name).to.equal('await');
+    expect(filter.async).to.equal(true);
+    expect(typeof filter.filter).to.equal('function');
+  });
```

Comment:
Yes definitly, will be the same for the plugins, but I'd like to chat with you first to make sure we have a good re-usable pattern for it across the monorepo. Unless you have any objections I'll merge it in like this and we can come back to it once we have an good approach figured out.